style: apply /style-guide pass to models/runs (#2724)
wandb/docs: models/runs/compare-runs.mdx

  <Frame>
  ![Baseline run details displayed prominently in a line plot](/images/models/pinned-and-baseline-runs/line-plot-baseline-run-foreground.png)
  </Frame>
- Hover over another run's legend label to display that run's line prominently and compare it with the baseline, which appears as a heavy dashed line. Lines for other visible runs appear with reduced saturation.
  <Frame>
  ![Another run compared with the baseline in a line plot](/images/models/pinned-and-baseline-runs/line-plot-baseline-comparison.png)
  </Frame>

### Summary metric deltas

When you set a run as the baseline, by default every other run that logs the same summary metric shows the delta from the baseline. The delta (amount of change) appears to the right of the metric's value in the run's row in the runs table.

By default, the delta appears as dark gray text on a dark gray background. To turn on semantic coloring for quick visual reference, set the **Metric directionality** for a column. With directionality set:

- If the other run **outperforms** (is directionally better than) the baseline, the delta appears as dark red text with a light red background.

> - If the other run **underperforms** (is directionally better than) the baseline, the delta appears as dark teal text with a light teal background.

Author: Fixed: underperforms now reads "(is directionally worse than)" the baseline.

3. Click the **action (<Icon icon="ellipsis" iconType="solid"/>)** menu, then select **Replace baseline**. <Note>If the menu item is inactive, ensure that you have at least one pinning slot available. If necessary, unpin a pinned run by clicking the circular pin icon next to a pinned run.</Note>
4. The new run becomes the baseline, and W&B automatically pins the previous baseline so you can find it. Optionally, unpin it by clicking its pin icon.

### Remove the baseline designation

To remove the baseline designation:

1. Navigate to your workspace.
2. In the run selector or runs table, find the current baseline run.
3. Click the **action (<Icon icon="ellipsis" iconType="solid"/>)** menu, then select **Remove baseline**. <Note>If the menu item is inactive, ensure that you have at least one pinning slot available. If necessary, unpin a pinned run by clicking the circular pin icon next to a pinned run.</Note>
4. W&B automatically pins the previous baseline so you can find it. Optionally, unpin it by clicking its pin icon.

### Compare runs to the baseline

After you set a baseline, you can compare other runs against it in line plots and the runs table.

> The baseline run is always visible in line plots for metrics the run has logged. In line plots, lines for the baseline run appear bolder than other lines.

Reviewer: ```suggestion
The baseline run stays visible in line plots for metrics it has logged. Its lines appear bolder than other runs.
```

Author: Applied your suggested rewording.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

  <Frame>
  ![Tooltip showing details for all visible runs at a given point](/images/models/pinned-and-baseline-runs/line-plot-point-details.png)
  </Frame>
- Hover over the baseline run's legend label to display the line prominently. It appears as a heavy dashed line. Lines for other visible runs appear with reduced saturation.
  <Frame>
  ![Baseline run details displayed prominently in a line plot](/images/models/pinned-and-baseline-runs/line-plot-baseline-run-foreground.png)
  </Frame>
- Hover over another run's legend label to display that run's line prominently and compare it with the baseline, which appears as a heavy dashed line. Lines for other visible runs appear with reduced saturation.
  <Frame>
  ![Another run compared with the baseline in a line plot](/images/models/pinned-and-baseline-runs/line-plot-baseline-comparison.png)
  </Frame>

### Summary metric deltas

When you set a run as the baseline, by default every other run that logs the same summary metric shows the delta from the baseline. The delta (amount of change) appears to the right of the metric's value in the run's row in the runs table.

> By default, the delta appears as dark gray text on a dark gray background. To turn on semantic coloring for quick visual reference, set the **Metric directionality** for a column. With directionality set:

Reviewer: ```suggestion
When you set a run as the baseline, every other run that logs the same summary metric shows the delta from the baseline by default. The delta, or amount of change, appears to the right of the metric value in the run’s row in the runs table.

By default, the delta uses dark gray text and a dark gray background. To turn on semantic coloring for quick visual reference, set the Metric directionality for a column. When directionality is set:
```

Author: Applied your rewording; used a straight apostrophe to match the doc.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

- If the other run **outperforms** (is directionally better than) the baseline, the delta appears as dark red text with a light red background.
- If the other run **underperforms** (is directionally worse than) the baseline, the delta appears as dark teal text with a light teal background.

To set the directionality for a metric:

1. In the runs table, hover over the column heading for the metric.
2. Click the **action (<Icon icon="ellipsis" iconType="solid"/>)** menu that appears.
3. Set **Metric directionality** to **Higher values are best** or **Lower values are best**.

The following screenshot shows how the runs `nanochat-train-base` and `nanochat-train-mid` compare with the baseline run `nanochat-train`. Delta metrics appear for `TOTAL_TRAINING_TIME`, `TRAIN/DT`, and `TRAIN/GRAD_NORM`.
![Summary metric deltas compared to the baseline run](/images/models/pinned-and-baseline-runs/baseline_run_deltas.png)

## Hide summary metric deltas in a workspace

> By default, a workspace with a baseline run always displays summary metric deltas. To reduce visual clutter or focus on absolute values, you can hide them for a workspace:

Reviewer: ```suggestion
By default, a workspace with a baseline run always displays summary metric deltas. To hide summary metric deltas in a workspace:
```

Author: Applied — simplified the lead-in.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

        with wandb.init(
          # Replace with your team and project name
            project="hyperparameter-tuning",

>             entity="[TEAM-NAME]",

Reviewer: Same comment as the other PRs. Don't use square brackets since it looks like we expect a list as an input.

Author: Changed `[TEAM-NAME]` to `<team-name>` and updated the matching lead-in sentence. Left the other bracket placeholders as-is for now (your alert.mdx suggestion kept `[YOUR-TEAM]`); happy to do a full sweep if you want one.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
